include/fmt/format.h

```diff
@@ -901,7 +901,7 @@ template <typename T = void> struct FMT_EXTERN_TEMPLATE_API basic_data {
 // Maps bsr(n) to ceil(log10(pow(2, bsr(n) + 1) - 1)).
 // This is a function instead of an array to workaround a bug in GCC10 (#1810).
 FMT_INLINE uint16_t bsr2log10(int bsr) {
-  constexpr uint16_t data[] = {
+  static constexpr uint16_t data[] = {
```

Comment:
I'm worried that this is going to cause GCC to trip up in the same way that #1810 identified. Note that MSVC isn't the only compiler to do this silliness with the data array, GCC does, too: https://gcc.godbolt.org/z/nbffbs. clang is unaffected (only a difference in the mangling). So if it does break GCC, can this at least be static everywhere except for when `FMT_GCC_VERSION == 0`?

Reply:
> I'm worried that this is going to cause GCC to trip up in the same way that #1810 identified.

Could you check it on godbolt?

Reply:
Sorry. I thought I was unable to reproduce the original issue (obviously isolated since godbolt's been updated with a "fixed" fmtlib), so I thought maybe GCC 10 had been fixed.
Got it to repro, and, no: https://godbolt.org/z/Ps4fod GCC 10 won't issue the string-op-overflow warnings with the data array being static. `bsr2log10` also no longer requires always_inline to avoid the warning, as well. Which is just trivia, as there's no reason to remove it.